### PR TITLE
Cosmetic

### PR DESCRIPTION
**Pull request by mattearnshaw**
- [ ] Add coverage for whatever new functionality, to a JUnit test if it's backend, to a Casper Test if it's frontend
- [ ] Make sure both push and pr travis builds are passing before asking for a review

### CODE REVIEW

**Comment by borismarin** [comment]
with that amount of commits, it's more of a plastic surgery than just
'cosmetic'

2017-12-01 14:09 GMT-02:00 Matt Earnshaw <notifications@github.com>:

**Comment by mattearnshaw** [comment]
@borismarin lol, fixed

**Comment by tarelli** [comment]
@tarelli ready to review

File: src/main/webapp/js/components/interface/tutorial/Tutorial.less [hunk 1]
```diff
@@ -29,7 +29,6 @@
 	width: auto;
 	minheight: 0px;
 	maxheight: none;
-	height: 80%;
```
Comment (tarelli) [inline src/main/webapp/js/components/interface/tutorial/Tutorial.less]: @mattearnshaw this makes the arrows move :)

Comment (mattearnshaw) [reply src/main/webapp/js/components/interface/tutorial/Tutorial.less]: this is what I was trying to fix, as you can see there is cut-off and you can't scroll either. I'll try and make it so button are fixed and scroll works (without breaking other deployments)

tutorial

Comment (borismarin) [reply src/main/webapp/js/components/interface/tutorial/Tutorial.less]: Where is the code for this particular screenshot stored? Nothing to do with this issue, but the equation for g_K is slightly odd: the maximum conductance for K is weirdly typeset as a product of _g_ and max_K (whatever that is), while it should read as a single parameter - usually typeset as `\bar{g}_K` 
image

Comment (mattearnshaw) [reply src/main/webapp/js/components/interface/tutorial/Tutorial.less]: @borismarin https://github.com/tarelli/tutorials/blob/182d2952ab56440a6ef7bd45400550c3ec46fecf/1_hh_practical/eq1.png

Comment (tarelli) [reply src/main/webapp/js/components/interface/tutorial/Tutorial.less]: @borismarin Equations were lifted from @pgleeson doc btw there's probably a better way to visualizing them other than embedding pre-rendered images.

**Comment by mattearnshaw** [comment]
@tarelli I've put the fixed height back but allowed the tutorial message to scroll. buttons remain fixed

**Pull request merged**